recipes/vscode-jupyter/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs 12.14.1
+    - code-server >=3.9
+    - {{ compiler('cxx') }}
```

Comment:
Linux build fails when using `compiler('c')` instead of compiler('cxx').
OSX build succeeds with both compilers.

Reply:
Can you use both here then?

Reply:
Done